src/wp-includes/class-wp-theme.php

```diff
@@ -276,13 +284,15 @@ public function __construct( $theme_dir, $theme_root, $_child = null ) {
 				$this->errors = new WP_Error( 'theme_no_stylesheet', __( 'Stylesheet is missing.' ) );
 			}
 			$this->template = $this->stylesheet;
+			$this->is_block_theme();
```

Comment:
I don't think this check is needed here, we can just set it to `false` since the theme is missing anyway.
```suggestion
			$this->block_theme = false;
```